code/__DEFINES/research/anomalies.dm

```diff
@@ -15,3 +15,34 @@
 
 /// Chance of anomalies moving every process tick
 #define ANOMALY_MOVECHANCE 45
+
+/// Blacklist of parts which should not appear when bioscrambled, largely because they will make you look totally fucked up
+GLOBAL_LIST_INIT(bioscrambler_parts_blacklist, list(
+	/obj/item/bodypart/chest/larva,
+	/obj/item/bodypart/head/larva,
+	/obj/item/bodypart/leg/left/tallboy,
+	/obj/item/bodypart/leg/right/tallboy,
+))
+
+/// Blacklist of organs which should not appear when bioscrambled.
+/// Either will look terrible outside of intended host, give you magical powers, are irreversible, or kill you
+GLOBAL_LIST_INIT(bioscrambler_organs_blacklist, list (
```

Comment:
why not do a `typecacheof()`? will make any the global list init faster (i think?) and i doubt you want mr. poop dimension coder in the year 2027 to add the `/obj/item/organ/internal/heart/cursed/poop_dimension_edition` and have it be exempt from this nifty little list. safer to assume subtypes are evil than they are safe in a blacklist IMO

(we can always add a whitelist that readds it to the global list after we remove it :))